docs/central.md

```diff
@@ -279,16 +279,22 @@ The publishing process for Maven Central consists of several steps
 4. The staging repository is released
 5. All artifacts in the released repository will be synchronized to maven central
 
-By running the following Gradle task the plugin will take care of steps 1 to 3 automatically:
+The plugin will always do step 1 to 3 and when automatic releases are enabled also take
+care of step 4.
+
+Regardless of whether it is done automatically or manually after the staging repository is released
+the artifacts will be synced to Maven Central. This process takes 10-30 minutes and when it is completed
+the artifacts are available for download.
+
+### Automatic release
+
+Run the following to let the plugin handle all steps automatically:
 
 ```
-./gradlew publishAllPublicationsToMavenCentralRepository --no-configuration-cache
+./gradlew publishAllPublicationsToMavenCentralRepository closeAndReleaseRepository --no-configuration-cache
```

Comment:
I'll do a follow up where I add `releaseRepository` as an alias for this task and a new `publishAndReleaseToMavenCentral` that depends on both `publishAllPublicationsToMavenCentralRepository` and `releaseRepository`. That way this becomes a lot shorter.